lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -700,6 +700,7 @@ class BarTouchTooltipData with EquatableMixin {
     double? tooltipHorizontalOffset,
     double? maxContentWidth,
     GetBarTooltipItem? getTooltipItem,
+    GetBarTooltipColor? getTooltipColor,
```

Comment:
What if we remove `tooltipBgColor` completely?

Reply:
Yes we could remove it completely but it was kept considering backward compatibility.

Reply:
this is done. old api is removed. changelog is updated.